packages/contracts-bedrock/scripts/winddown-upgrade/prod/Winddown.s.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { console } from "forge-std/console.sol";
+import { Script } from "forge-std/Script.sol";
+
+import { Proxy } from "contracts/universal/Proxy.sol";
+import { L1ChugSplashProxy } from "contracts/legacy/L1ChugSplashProxy.sol";
+import { WinddownConstants } from "../WinddownConstants.sol";
+
+import { IBalanceClaimer, BalanceClaimer } from "contracts/L1/winddown/BalanceClaimer.sol";
+import { L1StandardBridge } from "contracts/L1/L1StandardBridge.sol";
+import { OptimismPortal } from "contracts/L1/OptimismPortal.sol";
+
+import { L2OutputOracle } from "../../../contracts/L1/L2OutputOracle.sol";
+import { SystemConfig } from "../../../contracts/L1/SystemConfig.sol";
+
+contract WinddownUpgrade is Script {
+    function run() public {
+        uint256 _deployerPk = vm.envUint("PRIVATE_KEY_PROXY_ADMIN");
```

Comment:
have we considered using keystores like we do in the boilerplate?

Reply:
can you provide the reference?, I can check if we can use it